models/training-tuning-scripts/fraud-detection-models/model.py

```diff
@@ -0,0 +1,201 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
We should symlink this file in the example directory. i.e. Keep this file as the real file and create a symlink to this file in the example directory.

Also add a comment about the symlink in the readme